@here/harp-mapview/lib/ConcurrentWorkerSet.ts

```diff
@@ -216,9 +229,10 @@ export class ConcurrentWorkerSet {
 
         // respond with all pending request
         this.m_requests.forEach(entry => {
-            entry.resolver(new Error("worker destroyed"));
+            entry.resolver("worker destroyed");
```

Comment:
It looks like there is inconsistency with some other parts of the code that it use resolver() call with `new Error()` object. (See lines 534, 470)

Reply:
Good catch. Fixed